content/en-us/chat/in-experience-text-chat.md

```diff
@@ -16,7 +16,7 @@ The in-experience text chat system consists of both mutable APIs that you can ex
 The in-experience text chat system provides the following mutable APIs:
 
 - `Class.TextChatService`: This singleton class is responsible for managing the overall chat system, including handling chat message filtering, moderation, and user permissions. Accessible from the server, it provides a set of methods and events that other text chat APIs or user actions can invoke through the chat delivery workflow.
-- `Class.TextChannel`:This server-side class represents a text chat channel and passes user-sent chat messages from the client to the server and displays them to other users based on permissions. You can use it to create, modify, and manage text channels in your experience. Additionally, you can create multiple text channels to group users together for chat purposes, such as allowing users to chat with their group members that are not visible to others.
+- `Class.TextChannel`: This server-side class represents a text chat channel and passes user-sent chat messages from the client to the server and displays them to other users based on permissions. You can use it to create, modify, and manage text channels in your experience. Additionally, you can create multiple text channels to group users together for chat purposes, such as allowing users to chat with their group members that are not visible to others.
```

Comment:
The phrasing of the first sentence makes me think that it's server-side only, but then I read that it passes chat messages from client-to-server... so now I'm thinking it must be accessible client-side as well.

Reply:
I hadn't read the paragraph, I just saw the no-space typo 🙃
Definitely incorrect to say TextChannels are 'server side classes' — they should be instantiated on the server, sure, but clients absolutely need to make use of them.